core/pom.xml

```diff
@@ -585,7 +585,6 @@
 		<dependency>
 			<groupId>commons-dbcp</groupId>
 			<artifactId>commons-dbcp</artifactId>
-			<version>1.4</version>
 			<scope>test</scope>
 		</dependency>
```

Comment:
Dbcp is niet nodig als dependency, kan hier helemaal verwijderd worden.

Reply:
Gedaan.